backend/User/views.py

```diff
@@ -79,7 +80,17 @@ def delete(self, request, user_id):
 			return Response(status=status.HTTP_404_NOT_FOUND)
```

Comment:
I know it's not part of this PR, but shouldn't we first check that the request is well-formed and only if it is log the user out/ send a 404?

Reply:
As far as I understand the methods used are already making sure user is logged in and also the owner. So the 404 should never even happen in this case 🤷‍♀️